test/components/TaskListTest.js

```diff
@@ -136,22 +101,8 @@ describe('TaskList', () => {
   })
 
   context('when the archive button is clicked', () => {
-    let archiveTime
-
     before(() => {
-      store.dispatch({ type: 'TASKS_SELECT', task: {
-        storageKey: 'pull-163031382',
-      } })
-
       TestUtils.Simulate.click(renderedDOM().querySelector('#archive-button'))
-      archiveTime = new Date()
-    })
-
-    after(() => {
-      store.dispatch({
-        type: 'TASKS_SELECT', task: { storageKey: 'pull-163031382' },
-      })
-      store.dispatch({ type: 'TASKS_RESTORE' })
```

Comment:
The store is thrown out and recreated after each test, so we don't have to clean anything up!